features/bootstrap/Context/StockIndicatorExport/Domain/StockIndicatorExportContext.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace Inviqa\Acceptance\Context\StockIndicatorExport\Domain;
+
+use Behat\Behat\Context\Context;
+use Inviqa\StockIndicatorExport\Domain\Model\Product;
+use Inviqa\StockIndicatorExport\Domain\Model\Product\Sku;
+use Inviqa\StockIndicatorExport\Domain\Model\Product\Stock;
+use Inviqa\StockIndicatorExport\Domain\Model\StockIndicator;
+use PHPUnit\Framework\Assert;
+
+class StockIndicatorExportContext implements Context
```

Comment:
I think in this context where we have used the `assertTrue()` assertion and then used the `sameAs()` method from the `exportedStockIndicator()` as one of the parameters, we could use the PhpUnit `assertEquals()`.

Reply:
hmm good idea!